students/chelsea_nayan/lesson04/kata.py

```diff
@@ -0,0 +1,50 @@
+# chelsea_nayan, UWPCE Python 210, Lesson04: Kata Fourteen: Tom Swift Under Milk Wood
+# This was difficult!
+
+import random # Used to pick a random word from the dictionary value list
+import re # Used to use findall()
+
+story_limit = 10000 # Sets the character limit so the new_story can't go on forever...
+text_file = 'sherlock_big.txt' # Chooses the text file to be read
+first_two_words = ["One", "night"] # Chooses the first two words for the trigram
+
+with open(text_file, 'r') as infile, open('new_story.txt', 'w') as outfile:
```

Comment:
It's best to avoid having the files open while you prepare your data; meaning prepare your new text and then write new file. Also, code would be more readable if you broke it up into smaller functions.